inc/nux/class-storefront-nux-admin.php

```diff
@@ -68,7 +68,7 @@ public function admin_notices() {
 			}
 
 			// Coming from the WooCommerce Wizard?
-			if ( wp_get_referer() && 'index.php?page=wc-setup&step=next_steps' === basename( wp_get_referer() ) && 'post-new.php' === $pagenow ) {
+			if ( wp_get_referer() && 0 === strpos( basename( wp_get_referer() ), 'index.php?page=wc-setup' ) && 'post-new.php' === $pagenow ) {
```

Comment:
because if you skip the WC OBW (you click not right now), the referer step is not `next_steps`, but I think we still want to enable the possibility of adding starter content, etc.

There's a comment in the commit details: https://github.com/woocommerce/storefront/pull/1190/commits/f6deb0721a9cff75545efa2c4a1ae10f60940e2e